src/app/Http/Controllers/Auth/VerificationController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use Illuminate\Http\Request;
+use Illuminate\Contracts\View\View;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\VerifyRequest;
+use Illuminate\Auth\Events\Verified;
+use Illuminate\Http\RedirectResponse;
+use App\Http\Requests\ResendVerificationRequest;
+
+class VerificationController extends Controller
+{
+     //Display the email verification notice.
```

Comment:
Use docbloc but don't include the @param or @return on it because you already assigned the types of your parameter and return on your method. Kind a redundant right??


https://docs.phpdoc.org/guide/getting-started/what-is-a-docblock.html